Loop/Views/Presets/NewCustomPreset.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  NewCustomPreset.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 2/26/25.
+//  Copyright © 2025 LoopKit Authors. All rights reserved.
+//
+
+import LoopAlgorithm
+import UIKit
+import LoopKit
+
+struct PresetScheduleRepeatOptions: OptionSet {
```

Comment:
Instead of defining this all manually, we could use `Calendar.main.weekdaySymbols`.

https://developer.apple.com/documentation/foundation/calendar/2293235-weekdaysymbols